packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -216,6 +216,12 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
               'as a multiplier of the default timeout (e.g. "2x"), '
               'or as the string "none" to disable the timeout entirely.',
         defaultsTo: '30s',
+      )
+      ..addFlag('enable-fractional-translation',
+        help: 'Enables fraction translation of engine composited layers in '
```

Comment:
Should this be hidden with `hide: !verboseHelp`?

Reply:
Done